frontend/src/components/AdequacyCensusCharts/AdequacyCensusCharts.tsx

```diff
@@ -52,22 +53,33 @@ export let AdequacyCensusCharts = withStore('adequacies', 'method')<Props>(({ se
       </tr>
       {
         censusGroups.map(censusGroup =>
-          adequacyRowByCensusGroup(censusGroup, populationByAdequacyByGroup[censusGroup])
+          adequacyRowByCensusGroup(censusGroup, populationByAdequacyByGroup[censusGroup], format)
         )
       }
     </StatsBox>
   </div>
 })
 
-function adequacyRowByCensusGroup(censusGroup: string, populationByAdequacy: PopulationByAdequacy) {
+function adequacyRowByCensusGroup(censusGroup: string, populationByAdequacy: PopulationByAdequacy, format: Format) {
   let totalPopulation = populationByAdequacy.reduce((a: number, b: number) => a + b)
+  if (format === 'Percentage') {
+    return (
+      <tr>
+        <td>{censusGroup}</td>
+        <td>{formatPercentage(100 * populationByAdequacy[0] / totalPopulation)}</td>
+        <td>{formatPercentage(100 * populationByAdequacy[1] / totalPopulation)}</td>
+        <td>{formatPercentage(100 * populationByAdequacy[2] / totalPopulation)}</td>
+        <td>{formatPercentage(100 * populationByAdequacy[3] / totalPopulation)}</td>
+      </tr>
+    )
+  }
   return (
     <tr>
       <td>{censusGroup}</td>
-      <td>{formatPercentage(100 * populationByAdequacy[0] / totalPopulation)}</td>
-      <td>{formatPercentage(100 * populationByAdequacy[1] / totalPopulation)}</td>
-      <td>{formatPercentage(100 * populationByAdequacy[2] / totalPopulation)}</td>
-      <td>{formatPercentage(100 * populationByAdequacy[3] / totalPopulation)}</td>
+      <td>{formatNumber(populationByAdequacy[0])}</td>
```

Comment:
lgtm other than the fact that it looks like some copy pasta. should we abstract a bit?